backend/backend/settings/base.py

```diff
@@ -185,66 +186,115 @@ def get_required_setting(
 CSRF_TRUSTED_ORIGINS = [WEB_APP_ORIGIN_URL]
 CORS_ALLOW_ALL_ORIGINS = False
 
-
-# Application definition
-SHARED_APPS = (
-    # Multitenancy
-    "corsheaders",
-    # For the organization model
-    "account",
-    # Django apps should go below this line
-    "django.contrib.admin",
-    "django.contrib.auth",
-    "django.contrib.contenttypes",
-    "django.contrib.sessions",
-    "django.contrib.messages",
-    "django.contrib.staticfiles",
-    "django.contrib.admindocs",
-    # Third party apps should go below this line,
-    "rest_framework",
-    # Connector OAuth
-    "connector_auth",
-    "social_django",
-    # Doc generator
-    "drf_yasg",
-    "docs",
-    # Plugins
-    "plugins",
-    "feature_flag",
-    "django_celery_beat",
-)
-
 if not check_feature_flag_status(FeatureFlag.MULTI_TENANCY_V2):
-    SHARED_APPS = ("django_tenants",) + SHARED_APPS
-
-TENANT_APPS = (
-    # your tenant-specific apps
-    "django.contrib.admin",
-    "django.contrib.auth",
-    "django.contrib.contenttypes",
-    "django.contrib.messages",
-    "django.contrib.staticfiles",
-    "tenant_account",
-    "project",
-    "prompt",
-    "connector",
-    "adapter_processor",
-    "file_management",
-    "workflow_manager.endpoint",
-    "workflow_manager.workflow",
-    "tool_instance",
-    "pipeline",
-    "platform_settings",
-    "api",
-    "prompt_studio.prompt_profile_manager",
-    "prompt_studio.prompt_studio",
-    "prompt_studio.prompt_studio_core",
-    "prompt_studio.prompt_studio_registry",
-    "prompt_studio.prompt_studio_output_manager",
-    "prompt_studio.prompt_studio_document_manager",
-    "prompt_studio.prompt_studio_index_manager",
-    "usage",
-)
+    SHARED_APPS = (
```

Comment:
@muhammad-ali-e why do we split as shared apps and tenant apps in v2 also? Won't they all be in a single schema?

Reply:
This logic is actually for V1, not V2. The else condition handles the V2 logic. I temporarily split the shared apps and V2 apps to better understand the V2 applications. We will combine them later.